canonical-web-and-design/ubuntu.com
# Allow html in engage header

## Done

- Allow html in the engage header template with `| safe`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage/nfv-sdn-openstack
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that there is no `<br />` like on live


## Issue / Card

Fixes #6019
